frontend/components/FeatureList.vue

```diff
@@ -0,0 +1,280 @@
+<template>
+    <div class="flex flex-col w-full space-y-10">
+        <div v-for="feature in features" :key="feature.name">
+            <TextBlock :title="feature.title">
+                <div class="mb-4">
+                    <span class="text-gray-700 dark:text-gray-200">URL:</span>
+                    <a v-if="feature.url" :href="feature.url"> {{ feature.url }} </a>
+                    <span v-else class="italic">Not yet implemented</span>
+                </div>
+                <h3 class="text-lg text-gray-700 font-semibold dark:text-gray-200 my-2">List of features:</h3>
+                <ul
+                    v-for="(point, index) in feature.features"
+                    :key="index"
+                    class="list-inside list-disc text-gray-700 dark:text-gray-200"
+                >
+                    <li :class="`Indent-${point.indent}`">{{ point.text }}</li>
+                </ul>
+            </TextBlock>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import TextBlock from '~/components/common/TextBlock.vue';
+
+export default Vue.extend({
+    components: {
+        TextBlock,
+    },
+    data() {
+        return {
+            features: [
```

Comment:
I think this is debatable and maybe you put some thought into this. 
As this feature list is pretty long I was wondering if it might make more sense to split it off into a separate file and to then export the const. Might make editing texts/UI easier in the future as we managed to separate it. 

What do you think?

Reply:
I actually thought the same thing but doubted myself 😅 The main reason I used this file structure instead of just writing the text in the template is to improve readability of the component, so I agree that moving the text outside of it would increase that even more.